.travis.yml

```diff
@@ -4,6 +4,9 @@ addons:
 language: java
 jdk:
 - oraclejdk8
+sudo: required
+services:
+  - docker
```

Comment:
Concerned about the increase in build time due to this change and starting a container for the test

Reply:
For this Grails issue https://github.com/grails/grails-core/issues/10964 I also need postgres and instead of adding Testcontainers to `grails3-functional-tests` I'm going to use an embedded pg server. The issue constains a sample application that uses it.
Nevertheless I think it is a good idea to start using testcontainers for Micronaut because we integrate with a lot of 3rd parties and having the possibility to have a real service for the tests instead of a mock one is better. I think starting a container like pg will only add a few seconds to the tests.

Reply:
@ilopmar I was watching @puneetbehl's screen while he was running the test and it took quite a long time

Reply:
I haven't used docker in travis before. I just thought it will only take a few seconds (<10) like when I run a pg docker container in my laptop

Reply:
FYI starting a PostgreSQL container takes a few seconds, nearly the same as with an embedded one :)

Sometimes (if you run Docker on Mac/Windows) it makes sense to give it more CPUs / memory because otherwise only some is given by default

And of course one can reuse the same instance, but that's up to the testing scenario :)